src/Package/Impl/Utilities/ViewUtilities.cs

```diff
@@ -80,9 +80,11 @@ public static class ViewUtilities {
         }
 
         public static void SaveFile(this ITextView textView) {
-            RunningDocumentTable rdt = new RunningDocumentTable(RPackage.Current);
-            string filePath = textView.TextBuffer.GetFilePath();
-            rdt.SaveFileIfDirty(filePath);
+            if (textView != null) {
+                RunningDocumentTable rdt = new RunningDocumentTable(RPackage.Current);
+                string filePath = textView.TextBuffer.GetFilePath();
```

Comment:
With the change above, `filePath` can be `null`. Will `SaveFileIfDirty` handle it?